src/pkg/build/sources/base_environment.go

```diff
@@ -237,6 +237,36 @@ fi
 PS1="Singularity> "
 export LD_LIBRARY_PATH PS1
 `
+
+	// Contents of /.singularity.d/env/99-runtimevars.sh
+	base99runtimevarsShFileContent = `#!/bin/bash
+# Copyright (c) 2017-2018, SyLabs, Inc. All rights reserved.
+#
+# This software is licensed under a customized 3-clause BSD license.  Please
+# consult LICENSE file distributed with the sources of this project regarding
```

Comment:
LICENSE -> LICENSE.md 
or @jmstover get angry

Reply:
Sorry.  This is not a new file though.  It was present in 2.x and already has a license associated with it.  Should I still change?  I can do whatever.

Reply:
@jmstover  is the best one to answer that